posts/2020-12-15-openliberty-realworld-example-app.adoc

```diff
@@ -131,6 +131,8 @@ public class UserDao {
 
 == Try it Out!
 
+You can pull the code in and play with it on your own machine. Our full implementation can be found on our link:https://github.com/OpenLiberty/openliberty-realworld-example-app[Open Liberty GitHub] where the README should cover how to build it locally. To view Conduit locally, I personally chose link:https://github.com/gothinkster/angular-realworld-example-app[Angular] as my frontend, and followed their instructions on starting it. Of course, you can choose whichever link:https://codebase.show/projects/realworld[frontend] you'd like!
```

Comment:
...where the README should cover how to build it locally.
-->
...where the README explains how to build it locally.

...you'd like
-->
...you like